registration/models.py

```diff
@@ -371,3 +408,267 @@ def send_activation_email(self, site, request=None):
                 email_message.attach_alternative(message_html, 'text/html')
 
         email_message.send()
+
+
+class SupervisedRegistrationManager(RegistrationManager):
+
+    def _activate(self, profile, get_profile):
+
+        if not profile.user.is_active and not profile.activated:
+            site = Site.objects.get_current()
+            self.send_admin_approve_email(profile.user, site)
+
+        # do not set ``User.is_active`` as True. This will be set
+        # when a site administrator approves this account.
+        profile.activated = True
+        profile.save()
+        if get_profile:
+            return profile
+        else:
+            return True
```

Comment:
should this return user? the docstring for `activate_user` doesn't expect a `True`, only `False` or a `User` (or `RegistrationProfile`) object